company-profiles/infracloud.md

```diff
@@ -0,0 +1,33 @@
+# InfraCloud Technologies
+
+## Company blurb
+
+InfraCloud Technologies is an open-source, cloud-native computing company specializing in building AI cloud infrastructure using Kubernetes and other open-source technologies. They assist organizations in modernizing applications and infrastructure, focusing on GPU cloud solutions and private cloud deployments.
+
+## Company size
+
+InfraCloud Technologies has approximately **225–235 employees**.
+
+## Remote status
+
+- **Remote-first policy**: InfraCloud Technologies operates with a remote-first approach, allowing employees to work from anywhere.
+
+## Region
+
+India
```

Comment:
The readme says USA, India - which is correct?

Reply:
The region in the readme and profile should be where remote employees can be hired. The detail about the office can be added to the profile under office locations